github-repositories.md

```diff
@@ -7,9 +7,9 @@ description: The following details a list of relevant Sygma repositories.
 
 
 | Repo                                                                        | Description                                                                                                                                                                                                               | Status                                                                   |
-| --------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------ |
+| --------------------------------------------------------------------------- |---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| ------------------------------------------------------------------------ |
 | [**sygma-solidity**](https://github.com/sygmaprotocol/sygma-solidity)       | Smart contracts used by Sygma bridge                                                                                                                                                                                      | Public repo, [audited](audits/); beta.                                   |
-| ****[**sygma-relayer**](https://github.com/sygmaprotocol/sygma-relayer)**** | Golang framework used to build bridges                                                                                                                                                                                    | Public repo, [audited](audits/); beta.                                   |
+| ****[**sygma-relayer**](https://github.com/sygmaprotocol/sygma-relayer)**** | Go-implemented relayer service                                                                                                                                                                                            | Public repo, [audited](audits/); beta.                                   |
```

Comment:
Not sure how to define relayers (@waymobetta if you have a better idea), but for sure, they are not framework (this was probably referring to chainbridge-core).

Reply:
Thanks Mak, your revision is fine for now and we can revisit wordage down the road